src/extras.jl

```diff
@@ -89,6 +89,14 @@ julia> cut(-1:0.5:1, 2, labels=["A", "B"])
  "B"
  "B"
 
+ julia> cut(-1:0.5:1, 2, labels=[-0.5, +0.5])
+ 5-element CategoricalArray{Float64,1,UInt32}:
+  -0.5
+  -0.5
+  0.5
+  0.5
+  0.5
```

Comment:
The doctests seems to be failing here. Maybe a whitespace issue?

Reply:
Indeed!